icongenie/lib/utils/validate-profile-object.js

```diff
@@ -21,13 +21,22 @@ const paramsSchema = Joi.object({
     Joi.number().integer().min(0)
   ).min(1).max(2),
 
+  splashscreenIconRatio: Joi.number().integer().min(0).max(100)
+}
+
+const colorParamsSchemaProfileSave = {
+  themeColor: Joi.string().pattern(/^[0-9A-Fa-f]{3}([0-9A-Fa-f]{3})?$/),
```

Comment:
We can create a function to createPatterns, this will simplify this a bit.
```js
const createColorPattern = requiresHash => new RegExp(`^${requiresHash ? '#' : ''}[0-9A-Fa-f]{3}([0-9A-Fa-f]{3})?$`)
```